wiggum_app/static/js/scatterplot.js

```diff
@@ -603,9 +603,10 @@ var UpdateMatrixFormat = function(matrix, vars, category, trend_type) {
  * @param colLabels - dependent used for columns labels.
  * @param category - groupfeat and subgroup.
  * @param trend_type - trend type.
+ * @param detail_view_type - detail view type.
  * @returns matrix - containing information for cells.
  */
-var UpdateLinearRegressionMatrixFormat = function(matrix, rowLabels, colLabels, category, trend_type) {
+var UpdateLinearRegressionMatrixFormat = function(matrix, rowLabels, colLabels, category, trend_type, detail_view_type) {
```

Comment:
Do you still use the trend type? or does detail view type replace trend_type?

Reply:
I use detail view type for interacting with detail view, but keep trend_type for interactive with result table.